backend/app/services/implementations/log_records_service.py

```diff
@@ -47,7 +47,7 @@ def to_json_list(self, logs):
                         "attn_to": log[5],
                         "note": log[6],
                         "tags": log[7],
-                        "building": log[8],
+                        "building_id": log[8],
```

Comment:
so this should still be building - we don't want to return the ID of the building, but the name of it instead to display

Reply:
resolved to be addressed later